src/guillo-gmi/actions/remove_items.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react'
+import {Confirm} from '../components/modal'
+import {useContext} from 'react'
+import {TraversalContext} from '../contexts'
+
+export function RemoveItems(props) {
+  const Ctx = useContext(TraversalContext)
+  const { items = [] } = props
+  const last = items[items.length - 1]['@name']
+  const itemsNames = items
+    .map(item => item['@name'])
+    .join(', ')
+    .replace(`, ${last}`, ` and ${last}`)
+
+  async function removeItems() {
+    await Promise.all(items.map(async item => {
```

Comment:
@jordic I want to ask you if there is a direct way to remove multiple items instead of using a Promise.all? Thanks 🙂

Reply:
On a raw guillotina none, but there's a module guillotina_batch that allows to batch multiple calls on a single request, anyway.. I have a plan to be able to detect this kind of module availability.. but it's something we can do in the future

Reply:
btw I don't think guillotina_batch works with G6